src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Compositor/StationaryCameraCalibrationDataProvider.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.SpectatorView
+{
+    public class StationaryCameraCalibrationDataProvider : MonoBehaviour
+    {
+        private INetworkManager networkManager;
+
+        private void Awake()
+        {
+            networkManager = GetComponent<INetworkManager>();
+            if (networkManager == null)
+            {
+                throw new MissingComponentException("Missing network manager component");
+            }
+
+            networkManager.Connected += NetworkManagerConnected;
+            if (networkManager.IsConnected)
+            {
+                SendCalibrationData();
+            }
+        }
+
+        private void OnDestroy()
+        {
+            networkManager.Connected -= NetworkManagerConnected;
+        }
+
+        private void NetworkManagerConnected(INetworkConnection obj)
+        {
+            SendCalibrationData();
+        }
+
+#if UNITY_EDITOR && UNITY_WSA
+        private void SendCalibrationData()
+        {
+            UnityCompositorInterface.GetCameraCalibrationInformation(out CompositorCameraIntrinsics compositorIntrinsics);
```

Comment:
>GetCameraCalibrationInformation [](start = 37, length = 31)

it seems like this should be a tryget or we should check if calibration information is available before making this call.